spykshrk/franklab/data_containers.py

```diff
@@ -575,7 +575,7 @@ def create_default(cls, df, sampling_rate, arm_coord, parent=None, **kwds):
         if parent is None:
             parent = df
 
-        return cls(df=df, sampling_rate=sampling_rate, arm_coord=arm_coord, parent=parent, **kwds)
+        return cls(data=df, sampling_rate=sampling_rate, arm_coord=arm_coord, parent=parent, **kwds)
```

Comment:
i changed this because the super (DayEpochTimeseries) __init__ looks for 'data' kwarg